roles/docker/tasks/cleanup.yml

```diff
@@ -12,3 +12,9 @@
   become: true
   with_items:
     - "{{ docker_api_port }}"
+
+- name: cleanup contiv-compose
```

Comment:
Just FYI: there are still more issues with this ansible. The goal of a restart is not correctly captured. Whenever we do a restart, it tries to do an os update. Not only does this consume time, but it also fails intermittently. When this happens, the whole cluster gets messed up because we have lot of tasks with the run_once option. When one node fails, it picks the next available node to run these run_once tasks (e.g.aci-gw service). However, these services do not get cleaned up upon a restart. Now you have multiple such services running where there should be only one.